### DISCUSSION TITLE
Hide or Remove Excess Launcher Menus

### DISCUSSION THREAD

**Discussion by xMonody** [wezterm/wezterm#5540] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

wezterm 20240203-110809-5046fc22

### Ask your question!

![e1101f3e617e7534768ec1e96e22fefa](https://github.com/wez/wezterm/assets/76483540/7c7d9994-d888-422c-a949-f32b3c37addf)
I only need my custom launcher menu.

**Comment by wez** [comment]
https://wezfurlong.org/wezterm/config/lua/keyassignment/ShowLauncherArgs.html will let you choose what shows up

**Reply by xMonody** [reply]
I understand. Thank you very much, Mr. Wez.
